packages/amplify-category-notifications/src/pinpoint-helper.ts

```diff
@@ -310,7 +310,7 @@ export const ensurePinpointApp = async (
   let pinpointApp: Partial<ICategoryMeta> | undefined;
   let resourceName;
   const amplifyMeta = context.exeInfo.amplifyMeta || stateManager.getMeta();
-  const envName = appEnvName || context.exeInfo.localEnvInfo.envName || stateManager.getCurrentEnvName();
+  const envName: string = appEnvName || context.exeInfo.localEnvInfo.envName || stateManager.getCurrentEnvName() || '';
```

Comment:
is fallback to `''` safe? would this change how this fails downstream ?

Reply:
Looks like it is. It is only referenced subsequently in functions where a fallback blank string will be handled safely.

https://github.com/sdstolworthy/amplify-cli/blob/9fc6e16163f65fa3cf5e0b495e15a7e2b96d0ebe/packages/amplify-category-notifications/src/pinpoint-helper.ts#L169

https://github.com/sdstolworthy/amplify-cli/blob/9fc6e16163f65fa3cf5e0b495e15a7e2b96d0ebe/packages/amplify-category-notifications/src/pinpoint-name.ts#L11